### ISSUE TITLE
TaxCalculation bug on 3.0.3

### ISSUE THREAD

**Issue by joergstick** [spryker/support#72]
Bad BUG in `Spryker\Zed\Tax\Business\Model\TaxCalculation`

```
protected function sumExpenseTaxes(QuoteTransfer $quoteTransfer)
   {
        $totalTaxAmount = 0;
        foreach ($quoteTransfer->getItems() as $expenseTransfer) {
            $totalTaxAmount += $expenseTransfer->getSumTaxAmount();
        }
​
        return $totalTaxAmount;
    }
```

 `foreach ($quoteTransfer->getItems()`
should be `foreach ($quoteTransfer->getExpenses()`

causes Total Tax amount to be double the value it should be

**Comment by the-colorant** [comment]
Added to backlog. I will post an update here once fixed.

**Comment by dereuromark** [comment]
This is fixed with [3.0.4](https://github.com/spryker/Tax/releases/tag/3.0.4). Thank you for reporting it.
